app/build.gradle

```diff
@@ -27,6 +27,8 @@ dependencies {
     implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'com.android.support:appcompat-v7:27.1.1'
     implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    implementation 'android.arch.persistence.room:runtime:1.1.1'
+    annotationProcessor 'android.arch.persistence.room:compiler:1.1.1'
```

Comment:
Mına jerde bir probel taslaǵan jaqsı hár zattıń library in otdelno otdelno. Comment penen Room dep jazıp qoysań odanda jaqsı.